src/routes/dispensary/add/+page.server.ts

```diff
@@ -1,7 +1,39 @@
+import supabase from "$lib/supabase/client";
+
 export const actions = {
   create: async ({ request }) => {
-    const data = await request.formData()
-    const formName = data.get("name")
-    return { success: true, name: formName }
-  }
-}
+    try {
+      const data = await request.formData();
+      const formName = data.get("name");
+      const formAddress = data.get("address");
+      const formCity = data.get("city");
+      const formState = data.get("state");
+      const formZip = data.get("zip");
+      const formStatus = data.get("status");
+      const formCreatedBy = data.get("createdBy");
+
+      const { data: insertData, error } = await supabase
```

Comment:
Glad you found the syntax to insert records and parse the output. This looks great.